src/js/gi/components/If.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class If extends React.Component {
```

Comment:
I seriously hate this `If` component, but it was a necessity when porting over legacy code.